examples/template-change.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+

```

Comment:
@tuopouk 
Thank you for submitting this app. It's a very good app for new Dash community member to learn how to update their graph templates. 
 
Can you please update all the IDs in this file to be unique IDs. For example, the 'template' id could be `template-change-x-template`. 
You can read more about [unique IDs here](https://github.com/AnnMarieW/dash-app-gallery/blob/main/CONTRIBUTING.md#unique-ids).